### ISSUE TITLE
Is this module still functional?

### ISSUE THREAD

**Issue by DdraigX** [taxilof/MMM-GoogleKeep#9]
I just tried to set this up and all I got was a blank screen on my MM.

I must be doing something wrong?  I'm new to MM so I'm not sure where to check any logs.

**Comment by gtrsvk** [comment]
Yes it is working on RaspberryPi for me. You have to log in through your browser first, then it should log in just fine after you launch MagicMirror
